### ISSUE TITLE
Diagrams

### ISSUE THREAD

**Issue by ericsnekbytes** [jupyter/docs-team-compass#16]
Identify and create some diagrams that would be helpful to have on subprojects sites.

**Comment by krassowski** [comment]
I guess it would be good to start with fixing the existing diagrams which no longer render before adding any new ones:
- https://github.com/ipython/ipython/issues/14213
- https://github.com/jupyter/jupyter/issues/700

**Comment by ericsnekbytes** [comment]
Absolutely. We can probably break these out into a new issue (and add any other broken diagrams we can find to it).
